src/components/WebexMeetingControl/WebexMeetingControl.jsx

```diff
@@ -39,6 +39,7 @@ function renderButton(sc, action, display, style, showText, asItem, autoFocus, t
   } = display;
   const isDisabled = display.state === MeetingControlState.DISABLED;
   const isActive = display.state === MeetingControlState.ACTIVE;
+  let ariaLabelText = text;
```

Comment:
Added this variable because we are using variable text to show button-text and we don't want to show (No Microphone, disabled, No Camera, disabled) as button-text when mic and video is disabled, we have to show (No microphone, No camera) as button-text